docs/reference/transform/apis/put-transform.asciidoc

```diff
@@ -125,9 +125,6 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=pivot-aggs]
 (Required, object)
 include::{docdir}/rest-api/common-parms.asciidoc[tag=pivot-group-by]
 
-`max_page_search_size`:::
```

Comment:
This setting has been deprecated and moved to the settings part. I therefore removed the old mention, this might not be best-practice.